kubernetes/rollup-chart/templates/configmap.yaml

```diff
@@ -34,6 +34,7 @@ data:
   genesis_alloc_address: "{{ .Values.genesisAllocAddress }}"
   evm_network_id: "{{ .Values.evmNetworkId }}"
   evm_chain_id: "{{ .Values.evmChainId }}"
+  cometbft_rpc_endpoint: "http://sequencer-service:26657"
```

Comment:
To avoid having to `export COMETBFT_RPC_ENDPOINT=$cometbft_rpc_endpoint` in all the scripts you could just rename these as `COMETBFT_RPC_ENDPOPINT: "http://sequencer-service:26657"`.

Reply:
This was done on purpose, because otherwise it is unclear who is ingesting the environment variables